javalib/src/main/scala/java/nio/GenTypedArrayBuffer.scala

```diff
@@ -82,7 +82,7 @@ private[nio] final class GenTypedArrayBuffer[B <: Buffer](
     val typedArray = _typedArray
     val pos = position
     val lim = limit
-    typedArray.set(typedArray.subarray(pos, lim))
+    typedArray.set(typedArray.subarray(pos, lim).asInstanceOf[scala.scalajs.js.Iterable[Nothing]])
```

Comment:
I am doing this to leapflog type checking error with less hassle.
Not sure this is fine.

Reply:
I've been playing with this issue, trying to fix up the definition of `TypedArray` to make it work, but it's getting out of hands.

Given that this is not the only problem (there are further compile errors after this), I suggest you revert the change I suggested earlier and go back to the state with the `// FIXME` comment and the wildcards. We might be able to fix it in 1.x with more drastic changes.

Reply:
@sjrd 
Reverted the change and left `FIXME` comment along with wildcard.